doc/env/bin/mkenv.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
```

Comment:
This fails on current ubuntu since there's no `python`. Maybe use `python3`?

Reply:
Oh, maybe: https://mesonbuild.com/Python-module.html#find_installation

Then:

```
  command : [python_exe, 'mkenv.py', '@SOURCE_ROOT@', '@BUILD_ROOT@/doc/env'],
```

Reply:
It seems there's [some autodetection](https://mesonbuild.com/Reference-manual_functions.html#find_program).  Despite those docs, though, on Ubuntu 20.04 I had to set `mkenv.py` (but not `dist.py`) executable anyway.

I think everything should work now.

Reply:
It works! But I had to install `python3.10-venv` and `libclang-dev` (this is needed to get the `libclang.so` symlink), then add the llvm lib area to `lib_search_dirs` in `conf.py`:

```python
else:
    lib_search_dirs = [
        '/usr/lib',
        '/usr/local/lib',
        '/usr/lib/llvm-15/lib',
    ]
for lib_dir in lib_search_dirs:
```

Perhaps we should just add llvm 10 to 15 to the list? Or perhaps there's an easy way to get the clang major version without using libclang?

Reply:
Updated `conf.py` to search `/usr/lib/llvm-*/lib`, and mentioned the dependencies in the contributing docs.